### PR TITLE
update the docs of deploying v1.3 cloud part

### PR DESCRIPTION
**Pull request by ls889**
Integrated deployment docs of cloudcore

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind test
> /kind failing-test
> /kind feature

 /kind documentation

**What this PR does / why we need it**:
I integrate the documentations of deploying cloud part.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by kevin-wangzefeng** [comment]
> 0.16s$ make lint
hack/make-rules/lint.sh
sed: can't read build/cloud/07-deployment.yaml: No such file or directory
Makefile:98: recipe for target 'lint' failed
make: *** [lint] Error 123
The command "make lint" exited with 2.

The failure looks weird. @daixiang0 anything relevant to golangci-lint?

**Comment by daixiang0** [comment]
@kevin-wangzefeng fixed by #1792

**Comment by ls889** [comment]
> @kevin-wangzefeng fixed by #1792

thanks for fixing.  Retry and CI build passed @kevin-wangzefeng

**Comment by ls889** [comment]
@kevin-wangzefeng  ok,  I will resolve after #1774 merged

File: build/cloud/README.md [hunk 1]
```diff
@@ -1,36 +1,5 @@
...
-```
-
-Last, base on the `08-service.yaml.example`, create your own service,
-to expose cloud hub to outside of k8s cluster, so that edge core can
-connect to.
\ No newline at end of file
+Please refer to the README.md in the directory v1.3, pre1.3 or ha.
+* For KubeEdge v1.3: [README.md](v1.3/README.md)
+* For KubeEdge pre v1.3: [README.md](pre1.3/README.md)
+* HA deployment: refer to [README.md](ha/README.md)
```
Comment (kevin-wangzefeng) [inline build/cloud/README.md:5]: It's confusing to reader what's the relationship between HA and v1.3, pre v1.3.

File: build/cloud/07-deployment.yaml [hunk 2]
```diff
@@ -1,57 +0,0 @@
-apiVersion: apps/v1
```
Comment (kevin-wangzefeng) [inline build/cloud/07-deployment.yaml:1]: no deployment.yaml for pre1.3?

File: build/cloud/README.md (hunk 1 above)
Comment (ls889) [reply build/cloud/README.md:5]: @kevin-wangzefeng yes, it's  confusing to beginners. I will explain the overview of the three versions.

File: build/cloud/07-deployment.yaml (hunk 2 above)
Comment (ls889) [reply build/cloud/07-deployment.yaml:1]: @kevin-wangzefeng I'm sorry that maybe it's deleted by mistake, I will add it now.

File: build/cloud/README.md [hunk 3]
```diff
@@ -1,36 +1,11 @@
...
 
-This method will guide you to deploy the cloud part into a k8s cluster,
-so you need to login to the k8s master node (or where else if you can
-operate the cluster with `kubectl`).
+If you want to know more, please refer to [CHANGELOG-1.3.md](https://github.com/kubeedge/kubeedge/blob/master/CHANGELOG-1.3.md#kubeedge-v13-release-notes)
 
-The manifests and scripts in `github.com/kubeedge/kubeedge/build/cloud`
-will be used, so place these files to somewhere you can kubectl with.
+* For KubeEdge v1.3, it supports **CloudCore HA** and **EdgeNode auto TLS Bootstrapping** and so on:
+  * Single CloudCore: Refer to [README.md](v1.3/README.md)
```
Comment (kevin-wangzefeng) [inline build/cloud/README.md:7]: Shall we keep the main mode files under build/cloud directly?
Please note that https://github.com/kubeedge/kubeedge/pull/1781 is relying the yaml files

Comment (ls889) [reply build/cloud/README.md:7]: @kevin-wangzefeng  ok, I will modify base on #1781

File: build/cloud/README.md [hunk 4]
```diff
@@ -7,34 +7,41 @@ operate the cluster with `kubectl`).
...
-```bash
-cd build/cloud
-../tools/certgen.sh buildSecret | tee ./06-secret.yaml
+```yaml
+modules:
+      cloudHub:
+        advertiseAddress:
+         - 10.10.102.242
 ```
+How to know its IP in advance? You can label a node for running CloudCore and use `nodeSelector`.
```
Comment (kevin-wangzefeng) [inline build/cloud/README.md:26]: It's a bit confusing that why labeling a node would help get the IP address. It seems the description is incomplete.

Comment (GsssC) [reply build/cloud/README.md:26]: Just delte this option, func **NewDefaultCloudCoreConfig** will auto fill it with hostip.

**Comment by ls889** [comment]
@kevin-wangzefeng  if cloudcore don't have proxy. we need not to fill. it will get the correct IP when cloudcore runs.

File: build/cloud/README.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-##  Deploy the cloud part into a k8s cluster
+##  Deploy the cloud v1.3 part into a k8s cluster
```
Comment (fisherxu) [inline build/cloud/README.md:1]: 1.3?

File: build/cloud/01-prepare.yaml [hunk 6]
```diff
@@ -0,0 +1,608 @@
...
+                        type: string
+                    required:
+                      - value
+                    type: object
+                required:
+                  - propertyName
+                type: object
+              type: array
+          type: object
+  version: v1alpha1
```
Comment (fisherxu) [inline build/cloud/01-prepare.yaml:282]: Please update to v1alpha2, copy the yaml from build/crds

File: build/cloud/01-prepare.yaml [hunk 7]
```diff
@@ -0,0 +1,608 @@
...
+                    description: 'Required: The device property name to be accessed.
+                      This should refer to one of the device properties defined in
+                      the device model.'
+                    type: string
+                required:
+                  - propertyName
+                type: object
+              type: array
+          type: object
+  version: v1alpha1
```
Comment (fisherxu) [inline build/cloud/01-prepare.yaml:509]: ditto

File: build/cloud/01-prepare.yaml (hunk 6 above)
Comment (ls889) [reply build/cloud/01-prepare.yaml]: > Please update to v1alpha2, copy the yaml from build/crds

Thanks for reminding. done

**Comment by daixiang0** [comment]
@ls889 please rebase it

**Comment by ls889** [comment]
@daixiang0  hello, maybe i could close the pr. because another pr has been merged.

**Pull request closed without merging**